cmds/ocm/commands/ocicmds/common/handlers/artifacthdlr/closure.go

```diff
@@ -52,7 +52,7 @@ func traverse(hist common.History, o *Object, octx out.Context) []output.Object
 					UniformRepositorySpec: o.Spec.UniformRepositorySpec,
 					ArtSpec: oci.ArtSpec{
 						Repository: o.Spec.Repository,
-						Digest:     &ref.Digest,
+						ArtVersion: oci.ArtVersion{Digest: &ref.Digest},
```

Comment:
I think this version should be a pointer everywhere we reference it.

Reply:
see above

Reply:
What a mess :D